test/cli/use-strict.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const Path = require('path');
+
+const { test } = require('tap');
+
+const startCLI = require('./start-cli');
+
+test('for whiles that starts with strict directive', (t) => {
```

Comment:
Added this test for documentation purposes and to make sure that adding a strict directive doesn't *actually* break everything.